style.css

```diff
@@ -528,3 +517,604 @@ textarea {
+      margin: 0 16px;
+    }
+
+    .tags {
+      height: 8vh;
+      width: 100%;
+      padding-left: 6px;
+    }
+
+    .project,
+    .project1 {
+      width: 100%;
+    }
+
+    .button {
+      background-color: #fff;
+      color: #396df2;
+      padding: 12px;
+      text-align: center;
+      text-decoration: none;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 17px;
+      line-height: 24px;
+      margin: 20px 16px;
+      cursor: pointer;
+      border: 1px solid #6070ff;
+      border-radius: 8px;
+    }
+
+    .ruby {
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 16px;
+      color: #6070ff;
+      border-radius: 8px;
+      width: 110px;
+      height: 24px;
+      background: #ebebff;
+      padding: 4px 12px 4px 12px;
+      margin: 8px;
+      justify-content: center;
+    }
+
+    .base {
+      display: flex;
+    }
+
+    .form-content-container {
+      padding: 100px 250px 0 250px;
+    }
+
+    .form-header {
+      display: flex;
+      justify-content: center;
+      text-align: center;
+      font-family: 'Poppins', sans-serif;
+      width: 85%;
+      font-size: 40px;
+      font-weight: 700;
+      line-height: 52px;
+      color: #fff;
+      margin-top: 87px;
+    }
+
+    .form-description {
+      transition: transform 2s ease-in-out;
+      width: 100%;
+      margin: 12px 0;
+      font-family: 'Poppins', sans-serif;
+      font-size: 20px;
+      font-weight: 400;
+      line-height: 28px;
+      letter-spacing: 0;
+      text-align: center;
+      color: #ebebff;
+    }
+
+    form {
+      display: flex;
+      flex-direction: column;
+      width: 85%;
+      padding-left: 25px;
+      gap: 20px;
+    }
+
+    .form-element,
+    textarea {
+      background: #fff;
+      border-radius: 2px;
+      border: 1px solid #cfd8dc;
+      font-family: 'Poppins', sans-serif;
+      font-size: 17px;
+      font-weight: 400;
+      line-height: 20px;
+      letter-spacing: 0;
+      text-align: left;
+      padding: 15px 16px;
+    }
+
+    textarea {
+      height: 170px;
+      width: 100%;
+      margin-top: -2px;
+      margin-bottom: 27px;
+      color: #000;
+      font-size: 15px;
+      line-height: 24px;
+    }
+
+    .form-element {
+      width: 100%;
+      height: 48px;
+      color: #172b4d;
+    }
+
+    .submit-btn-container {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+
+    .about-text {
+      width: 90%;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 16px;
+      line-height: 24px;
+      color: #344563;
+      margin-left: 25px;
+      padding-top: 16px;
+    }
+
+    .about-base {
+      width: 100%;
+      margin-top: 15%;
+    }
+
+    .language-desktop {
+      width: 100%;
+      margin-top: 100px;
+    }
+
```

Comment:
Please make sure your `about-myself` and `language-desktop` are `display-flex` and their `flex-direction` are `row` to make side by side as Figma design.
Your design
![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/101523582/6a996413-2bfe-45fb-b676-e63745859724)

Figma design
![image](https://github.com/JoyTaribagshaw/My-Portfolio/assets/101523582/5ef9481a-9f1c-497d-82b0-4ebc4faa5c7d)

- [ ] Kindly, decrease the social media icons size too.

Reply:
Thank you for the feedback. I have resolved this issue.
![MicroverseScreenshot3](https://github.com/JoyTaribagshaw/My-Portfolio/assets/111312788/4a1bb596-d733-4ccf-b5e4-af1d7db4d949)